File: operationsOnAnArray.cpp
#include <bits/stdc++.h>
using namespace std;
#define endl "\n"

int main(void) {
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);
    
    int n;
    cin >> n;
    vector<long> v(n);
    for (int i = 0; i < n; i++)
        cin >> v[i];
    sort(v.begin(), v.end());
    long ans = LONG_MAX;
    for (int i = 0; i < n; i++) {
        long temp = 0;
        for (int j = 0; j < n; j++)
            temp += min(abs(v[i] - v[j]), v[j]);
        ans = min(ans, temp);
    }
    cout << ans << endl;
}

File: operationsOnAnArray.py
n = int(input())
ls = list(map(int, input().split()))
ans = 1e18
for i in range(n):
	temp = 0
	for j in range(n):
		temp += min(abs(ls[i] - ls[j]), ls[j])
	ans = min(ans, temp)
print(ans)
